src/main/java/core/basesyntax/NumberSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class NumberSupplier {
+
+    private static final Random RANDOM = new Random();
+    private static final int BOUND = 100;
+
+    public int getRandomNumber() {
+        return RANDOM.nextInt(BOUND);
+    }
```

Comment:
```suggestion
    private final Random random = new Random();

    public int getRandomNumber(int bound) {
        return RANDOM.nextInt(bound);
    }
```

You can leave this implementation, just as a recommendation: you can also pass bound as an incoming parameter, so you can use your NumberSupplier to generate random numbers with different bounds 🙂